setup.cfg

```diff
@@ -36,7 +36,7 @@ packages = find:
 python_requires = >=3.7
 install_requires =
     babel
-    entrypoints>=0.2.2
+    importlib_metadata>=1.4;python_version=="3.7"
```

Comment:
unfortunately, the situation isn't quite this simple, as the `group` argument wasn't introduced until [3.10](https://docs.python.org/3.10/library/importlib.metadata.html#entry-points) and wasn't added until 3.6 of [importlib-metadata](https://importlib-metadata.readthedocs.io/en/latest/using.html#entry-points).

So unfortunately, the safest  thing to do at this point is more like:

```ini
install_requires =
    importlib_metadata>=3.6;python_version<3.10
```

and then

```py
import sys

if sys.version_info < (3,10):
    from importlib_metadata import entry_points 
else:
    from importlib.metadata import entry_points 

# ...
    entry_points(group=JUPYTERLAB_LANGUAGEPACK_ENTRY)
```

Reply:
Thanks @bollwyvl